.github/workflows/ccpp-build.yml

```diff
@@ -0,0 +1,26 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        build-type: [Debug, Release, RelWithDebInfo]
+    steps:
+    - uses: actions/checkout@v2
+    - name: update
+      run: sudo apt-get update -y
+    - name: Get Packages
+      uses: mstksg/get-package@v1
+      with:
+        apt-get: uuid-dev libjson-c-dev libhwloc-dev lcov libtbb-dev
+    - name: configure ${{ matrix.build-type }}
+      run: mkdir ${{ github.workspace }}/.build && cd ${{ github.workspace }}/.build && cmake .. -DCMAKE_BUILD_TYPE=${{ matrix.build-type }}
+    - name: make ${{ matrix.build-type }}
```

Comment:
@r-rojo do you see value in generating `deb` packages in same flow?

I only see `ubuntu` supported by GitHub's default virtual environments, so unfortunately can't natively extend functionality to `centos`/`fedora` VMs.

Reply:
I don't recall travis supporting centos/fedora either. These are just sanity build checks. We can/should revisit packaging later though.

Reply:
ack